src/model/test/AirLoopHVACOutdoorAirSystem_GTest.cpp

```diff
@@ -27,76 +27,451 @@
-//
-//  Node supplyOutletNode = airLoop.supplyOutletNode();
-//  outdoorAirSystem.addToNode(supplyOutletNode);
-//  coil.addToNode(supplyOutletNode);
-//  boost::optional<ModelObject> testObject = airLoop.supplyComponent(outdoorAirSystem.handle());
-//  ASSERT_TRUE(testObject);
-//  EXPECT_EQ(outdoorAirSystem, *testObject);
-//  std::vector<HVACComponent> edges = outdoorAirSystem.getImpl<detail::HVACComponent_Impl>()->edges(false); // should be Node
-//  ASSERT_EQ(1, edges.size());
-//  edges = edges[0].getImpl<detail::HVACComponent_Impl>()->edges(false); // should be CoilHeatingElectric
-//  ASSERT_EQ(1, edges.size());
-//  EXPECT_EQ(coil, edges[0]);
-//}
+TEST_F(ModelFixture, AirLoopHVACOutdoorAirSystem_Clone_SeveralERVs) {
+
+  // Add the OASystem to AirLoopHVAC and save resulting OSM to check the clones in OSApp
+  constexpr bool debug = true;
+
+  Model m;
+
+  ControllerOutdoorAir controller(m);
+  AirLoopHVACOutdoorAirSystem oaSystem(m, controller);
+
+  ASSERT_TRUE(oaSystem.outboardReliefNode());
+  auto outboardReliefNode = oaSystem.outboardReliefNode().get();
+  auto outboardOANode = oaSystem.outboardOANode().get();
+  outboardOANode.setName("(OA)");
+  outboardReliefNode.setName("(RE)");
+
+  CoilHeatingElectric oaCoilA(m);
+  oaCoilA.setName("OACoilA");
+  oaCoilA.setComment("OACoilA");
+  EXPECT_TRUE(oaCoilA.addToNode(outboardOANode));
+
+  CoilHeatingElectric reliefCoilA(m);
+  reliefCoilA.setName("RECoilA");
+  reliefCoilA.setComment("RECoilA");
+  EXPECT_TRUE(reliefCoilA.addToNode(outboardReliefNode));
+
+  HeatExchangerAirToAirSensibleAndLatent ervA(m);
+  ervA.setName("ERVA");
+  ervA.setComment("ERVA");
+  EXPECT_TRUE(ervA.addToNode(outboardOANode));
+
+  CoilHeatingElectric oaCoilB(m);
+  oaCoilB.setName("OACoilB");
+  oaCoilB.setComment("OACoilB");
+  EXPECT_TRUE(oaCoilB.addToNode(outboardOANode));
+
+  HeatExchangerAirToAirSensibleAndLatent ervB(m);
+  ervB.setName("ERVB");
+  ervB.setComment("ERVB");
+  EXPECT_TRUE(ervB.addToNode(outboardOANode));
+
+  CoilHeatingElectric reliefCoilB(m);
+  reliefCoilB.setName("RECoilB");
+  reliefCoilB.setComment("RECoilB");
+  EXPECT_TRUE(reliefCoilB.addToNode(outboardReliefNode));
+
+  EXPECT_EQ(18, oaSystem.components().size());       // ERV counted twice
+  EXPECT_EQ(9, oaSystem.reliefComponents().size());  //     o-----RECoilA-----o-----ERVA-----o-----ERVB-----o-----RECoilB-----(RE)
+  EXPECT_EQ(9, oaSystem.oaComponents().size());      //  (OA)-----ERVB-----o-----OACoilB-----o-----ERVA-----o-----OACoilA-----o
+                                                     //  <=>
+                                                     //  (RE)---- RECoilB-----o----ERVB-----o-----------------------ERVA-----o-----RECoilA-----o
+                                                     //                             X                                 X
+                                                     //  (OA)----------------------ERVB-----o-----OACoilB-----o-----ERVA-----o-----OACoilA-----o
```

Comment:
More complex test with two ERVs to ensure we do end up with the correct layout after.